randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -152,8 +152,12 @@ Extra - room_id: [3]
       All of the following:
           Morph Ball and Speed Booster
           Any of the following:
-              Shinespark Tricks (Advanced) or Can Use Bombs
-              Can Bounce in Ball and Can Use Any Bombs
+              Any of the following:
+                  # No shinespark trick to create a morph tunnel across the top.
+                  Can Use Bombs
+                  Can Bounce in Ball and Can Use Any Bombs
+              # Trick to break the front speedblocks to leave a tunnel to get to the bomb block without needing a bounce item.
+              Shinespark Tricks (Advanced) and Can Use Any Bombs
```

Comment:
I'm not sure if this is the best way to write it. I feel like this is tricky. It might be best to have the spark tunnel just be for power bombs since if you have bombs you can just bomb jump to the hole anyway and you wouldn't need to do the shinespark trick in that event. Where as the other one can be either bombs or power bombs and the ability to bounce. I think changing any bombs to power bombs mightbe the move. Will actually just make the change now.